mlflow/tracking/_model_registry/fluent.py

```diff
@@ -24,13 +24,17 @@ def register_model(model_uri, name):
         client.create_registered_model(name)
```

Comment:
how about we add a eprint after this like `"Creating registered model %s" % name` to match the one about the existing model below?

Reply:
Done, thanks!